### PR TITLE
docs: fix links and format for mongodbatlas secrets

### PR DESCRIPTION
**Pull request by fairclothjm**
Backporting to 1.18.x so that the changes go live

### CODE REVIEW

File: website/content/docs/secrets/mongodbatlas.mdx [hunk 1]
```diff
@@ -2,49 +2,45 @@
...
 
-The MongoDB Atlas Secrets Engine generates Programmatic API keys. The created MongoDB Atlas secrets are
+The MongoDB Atlas secrets engine generates Programmatic API keys. The created MongoDB Atlas secrets are
 time-based and are automatically revoked when the Vault lease expires, unless renewed.
 
 Vault will create a Programmatic API key for each lease that provide appropriate access to the defined MongoDB Atlas
 project or organization with appropriate role(s). The MongoDB Atlas Programmatic API Key Public and
 Private Keys are returned to the caller. To learn more about Programmatic API Keys visit the
-[Programmatic API Keys Doc](https://docs.atlas.mongodb.com/reference/api-docs/apiKeys/).
+[Programmatic API Keys Doc](https://docs.atlas.mongodb.com/configure-api-access/#programmatic-api-keys).
```
Comment (tvoran) [inline website/content/docs/secrets/mongodbatlas.mdx:17]: BTW this seems to redirect to https://www.mongodb.com/docs/atlas/configure-api-access/#programmatic-api-keys now

File: website/content/docs/secrets/mongodbatlas.mdx [hunk 2]
```diff
@@ -2,49 +2,45 @@
...
 
-  <Note>
-
-  The information below relates to the **MongoDB Altas secrets engine**. Refer to the
-  [MongoDB Atlas **database** secrets engine](/vault/docs/secrets/databases/mongodbatlas)
-  for information about using the MongoDB Atlas database plugin for the Vault
-  database secrets engine.
-
-  </Note>
+~> **NOTE: The information below relates to the **MongoDB Altas secrets engine**. Refer to the
```
Comment (tvoran) [inline website/content/docs/secrets/mongodbatlas.mdx:19]: I thought the `~> ` style is considered legacy now? https://github.com/hashicorp/vault/blob/fd96ac79b922aed4d9e25743e2a4cc05ba6fca81/website/README.md#L107

File: website/content/docs/secrets/mongodbatlas.mdx [hunk 3]
```diff
@@ -2,49 +2,45 @@
...
 
-1. Enable the MongoDB Atlas Secrets Engine:
+1. Enable the MongoDB Atlas secrets engine:
 
-   ```bash
+    ```shell-session
    $ vault secrets enable mongodbatlas
-   Success! Enabled the mongodbatlas Secrets Engine at: mongodbatlas/
+   Success! Enabled the mongodbatlas secrets engine at: mongodbatlas/
    ```
```
Comment (tvoran) [inline website/content/docs/secrets/mongodbatlas.mdx:34]: Looks like `shell-session` is out of line with the rest of the code block.

File: website/content/docs/secrets/mongodbatlas.mdx [hunk 4]
```diff
@@ -57,7 +53,7 @@ steps are usually completed by an operator or configuration management tool.
 1. Once you have a MongoDB Atlas Programmatic Key pair, as created in the previous step, Vault can now
    be configured to use it with MongoDB Atlas:
 
-   ```bash
+    ```shell-session
    $ vault write mongodbatlas/config \
        public_key=yhltsvan \
        private_key=2c130c23-e6b6-4da8-a93f-a8bf33218830
```
Comment (tvoran) [inline website/content/docs/secrets/mongodbatlas.mdx:59]: Looks like `shell-session` is out of line with the rest of the code block.

File: website/content/docs/secrets/mongodbatlas.mdx (hunk 2 above)
Comment (fairclothjm) [reply website/content/docs/secrets/mongodbatlas.mdx:19]: I don't know, is it? Is the Note tag the preferred way now?

File: website/content/docs/secrets/mongodbatlas.mdx (hunk 1 above)
Comment (fairclothjm) [reply website/content/docs/secrets/mongodbatlas.mdx]: Thanks! Updated all links to the new domain

File: website/content/docs/secrets/mongodbatlas.mdx (hunk 2 above)
Comment (fairclothjm) [reply website/content/docs/secrets/mongodbatlas.mdx]: I went ahead and changed them all to use the Note tag. Let me know if there is another preferred format.

**Pull request merged**